authui/src/authflowv2/customSelect.ts

```diff
@@ -29,7 +29,7 @@ export class CustomSelectController extends Controller {
   declare readonly inputTarget: HTMLInputElement;
   declare readonly triggerTarget: HTMLButtonElement;
   declare readonly dropdownTarget: HTMLElement;
-  declare readonly searchTarget: HTMLInputElement;
+  declare readonly searchTarget: HTMLInputElement | undefined;
```

Comment:
target cannot be undefined. If you try to access a target which doesn't exist, is is an error.
We need to use hasXTarget to check for existence.
https://stimulus.hotwired.dev/reference/targets#optional-targets